### PR TITLE
pythonPackages.schema: Fix

### PR DESCRIPTION
**Pull request by tobim**
This depends on contextlib2 and mock. The preConfigure hook
can probably be removed with the next version bump, see
https://github.com/keleshev/schema/pull/222.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/development/python-modules/schema/default.nix [hunk 1]
```diff
@@ -10,11 +10,19 @@ buildPythonPackage rec {
     sha256 = "c9dc8f4624e287c7d1435f8fd758f6a0aabbb7eff442db9192cd46f0e2b6d959";
   };
 
-  checkInputs = [ pytest ];
+  preConfigure = ''
+    sed -i 's/==/>=/' requirements.txt
```
Comment (FRidh) [inline pkgs/development/python-modules/schema/default.nix]: please use substituteInPlace instead

Comment (tobim) [reply pkgs/development/python-modules/schema/default.nix]: Done.

**Comment by FRidh** [comment]
@GrahamcOfBorg build pythonPackages.schema

**Review by jonringer — APPROVED** [review]
`nix-review` passes on NixOS
diff LGTM
fixes broken build

```
[7 built (1 failed), 15 copied (12.7 MiB), 6.1 MiB DL]
error: build of '/nix/store/439xqsc9by2i95wkaqwyi12p23mnybir-env.drv' failed
https://github.com/NixOS/nixpkgs/pull/72690
1 package failed to build:
dvc-with-remotes

7 package were build:
dvc python27Packages.internetarchive python27Packages.schema python37Packages.internetarchive python37Packages.schema python38Packages.internetarchive python38Packages.schema
```

**Pull request merged**